docker/ccf.proxy.yaml

```diff
@@ -35,5 +35,4 @@ services:
     environment:
       http_proxy: "${http_proxy}"
       https_proxy: "${https_proxy}"
-      no_proxy: "${no_proxy},pdo-tp-ccf,pdo-build"
-
+      no_proxy: "${no_proxy},pdo-tp-ccf,pdo-build,ccf-pdo-build"
```

Comment:
the ccf container AFAIK does not make any outgoing call, it shouldn't need anything other than pdo-tp-ccf  (itself) as part of no_proxy